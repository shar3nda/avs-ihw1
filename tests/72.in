7
9 9 10 4 10 3 10