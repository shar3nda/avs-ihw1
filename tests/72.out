-4 0 -6 0 10 2 7 -3 2 8